storage/rinchi/src/test/java/org/openscience/cdk/rinchi/KeyBase26Test.java

```diff
@@ -15,34 +46,141 @@ void getBase26Triplet_2703_DZZ() {
     void getBase26Triplet_2704_FAA() {
         assertEquals("FAA", KeyBase26.getBase26Triplet(2704));
     }
-    
+
     @Test
-    void getBase26Triplet_2704_FaAA() {
-        assertEquals("FAA", KeyBase26.getBase26Triplet(16383));
+    void getBase26Triplet_2704_ZZZ() {
```

Comment:
Could please change the method name so it states 16383 instead of 2704. 
I've missed that.

Reply:
✅